### DISCUSSION TITLE
Implement WebView as a Theia Widget

### DISCUSSION THREAD

**Discussion by sboudreaux96** [eclipse-theia/theia#9592] [Q&A]
Hello - I have noticed that a web view plugin extension is offered in Theia under the following path: theia/packages/plugin-ext/src/main/browser/webview/webview.ts and I wanted to know what the easiest way would be to implement this as a Theia widget? Any help would be appreciated!

**Comment by vince-fugnitto** [comment]
@sboudreaux96 I do not believe it is currently supported to use `webviews` outside of plugins: https://github.com/eclipse-theia/theia/issues/6562.

**Reply by sboudreaux96** [reply]
@vince-fugnitto thank you for the quick response! So what would be the use of webviews as a plugin? Are there examples of this?

**Reply by vince-fugnitto** [reply]
> @vince-fugnitto thank you for the quick response! So what would be the use of webviews as a plugin? Are there examples of this?

You can take a look at the documentation of vscode, which also includes examples: https://code.visualstudio.com/api/extension-guides/webview.
